.github/workflows/test-gcp-dm.yml

```diff
@@ -80,12 +85,10 @@ jobs:
 
       - name: Install CSPM GCP integration
         id: cspm-gcp-integration
-        working-directory: deploy/test-environments/fleet_api
         env:
           STACK_VERSION: ${{ env.ELK_VERSION }}
           DEPLOYMENT_NAME: ${{env.GCP_DEPLOYMENT_NAME}}
         run: |
-          poetry install
           poetry run python src/install_cspm_gcp_integration.py
```

Comment:
This is my mistake; the path is incorrect. It should now be `tests/integrations_setup/install_cspm_gcp_integration.py`.